bee-ternary/src/b1t6.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{Btrit, RawEncoding, RawEncodingBuf, TritBuf, Trits, Tryte};
+
+use std::convert::TryFrom;
+
+const TRITS_PER_BYTE: usize = 2;
+const TRITS_PER_TRYTE: usize = 3;
+
+/// Decode a series of trits into bytes.
+pub fn decode(src: &Trits) -> Vec<u8> {
+    assert!(src.len() % TRITS_PER_BYTE == 0);
+    src.iter_trytes()
+        .zip(src[TRITS_PER_TRYTE..].iter_trytes())
+        .map(|(a, b)| decode_group(a as i8, b as i8).unwrap() as u8)
```

Comment:
Is this unwrap safe ?

Reply:
It's the same unwrap you had

Reply:
Yeah but I hacked this really quickly, we should probably handle it

Reply:
Otherwise it's quite easy to make the node crash

Reply:
Added error handling here